code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -0,0 +1,26 @@
+/mob/living/simple_animal/hostile/regalrat
+	name = "Regal Rat"
+	desc = "An evolved rat, created through some strange science. It leads nearby rats with deadly efficiency to protect its kingdom."
```

Comment:
ooh, what if you gave player-controlled regal rats the ability to command other mice to move to a location, sort of like what blobs can do with their blob spores?